.gitignore

```diff
@@ -24,6 +24,9 @@ __pycache__/
 *.py[cod]
 *$py.class
 timit_data/
+grid_generated*
+grid_ori*
```

Comment:
Let's add it to the gitignore file in the examples, because this is generated by the example, correct?

Reply:
Yes, but if you run this from the root, they are generated at the top. I think it is fine.